TaskDaemon.Core/src/me/aa07/paradise/taskdaemon/core/Core.java

```diff
@@ -121,11 +136,11 @@ private void setupJobs(Scheduler scheduler, DbCore dbCore, ConfigHolder config,
             .withSchedule(CronScheduleBuilder.cronSchedule("0 0 8 * * ?")) // Every day - 8AM
             .build();
 
-
         // Schedule all
         scheduler.scheduleJob(jd_bouncerrestart, ct_bouncerrestart);
         scheduler.scheduleJob(jd_ip2asn, ct_ip2asn);
         scheduler.scheduleJob(jd_profilercleanup, ct_profilercleanup);
+        scheduler.scheduleJob(jd_aclcleanup, ct_aclcleanup);
```

Comment:
```suggestion
        scheduler.scheduleJob(jd_aclcleanup, ct_aclcleanup);
        scheduler.scheduleJob(jd_bouncerrestart, ct_bouncerrestart);
        scheduler.scheduleJob(jd_ip2asn, ct_ip2asn);
        scheduler.scheduleJob(jd_profilercleanup, ct_profilercleanup);
```

Reply:
Diff looks awful but its just moving the AclCleanup stuff to the top of the list for alphabetisation